lang/en/lang.php

```diff
@@ -182,6 +182,7 @@
         'login_first' => 'You must be logged in first!',
         'already_active' => 'Your account is already activated!',
         'activation_email_sent' => 'An activation email has been sent to your email address.',
+        'activation_by_admin' => 'You have successfully registered. Your account is not yet active.',
```

Comment:
@mennax Would you mind making the same change to your Italian translation update as well? 
```suggestion
        'activation_by_admin' => 'You have successfully registered. Your account is not yet active and must be approved by an administrator.',
```